### PR TITLE
Feature/api connecting

### PR DESCRIPTION
**Pull request by hheyhhay**
Project 2 PR Template: Work it out 

What iteration is being worked on?
API Connecting *& updating user and UserRepositiry class

What is in request and being accomplished?

Updated User.js and UserRepository to return what is wanted
Added Intatiation of UserReposity and User classes with random index number
Rendered greetings to match name 

Collaborators: 
@jhortizu01 @hheyhhay

Other information:
- Need to refactor render function to use for each. 
-  need to make sure the random number is between 0 - 49 (based of indexes)

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -1,61 +1,65 @@
...
+//I don't think these need to be global
+let userDataArray;
+let sleepDataArray;
+let activityDataArray;
+let hydrationDataArray;
+
+let userRepo;
+let user;
+
+let bodySection = document.getElementById("body");
```
Comment (jhortizu01) [inline src/scripts.js:19]: label query selectors

File: src/scripts.js [hunk 2]
```diff
@@ -1,61 +1,65 @@
...
-let sleepDataArray = [];
-let activityDataArray = [];
-let hydrationDataArray = [];
 
 
 
 //✨ Functions ✨
-function fetchData() {
-  Promise.all([getData(), sleepData(), activityData(), hydrationData()]).then((values) =>
+function fetchData() { // this function should be in apiCalls apiCalls.userData().then((promise) => store user data, insantiate userRepo, then instantiate newUser)
```
Comment (jhortizu01) [inline src/scripts.js:30]: remove comment when refactoring

File: src/scripts.js [hunk 3]
```diff
@@ -1,61 +1,65 @@
...
-}
-
 
 
-console.log(userDataArray)
+  userDataArray = values[0].userData;
+  sleepDataArray = values[1].sleepData;
+  activityDataArray = values[2].activityData;
+  hydrationDataArray = values[3].hydrationData;
+  console.log(sleepDataArray)
```
Comment (jhortizu01) [inline src/scripts.js:42]: remove console.log during refactor

File: src/scripts.js (hunk 1 above)
Comment (hheyhhay) [reply src/scripts.js:19]: Thanks!

File: src/scripts.js (hunk 2 above)
Comment (hheyhhay) [reply src/scripts.js:30]: Okay!

File: src/scripts.js (hunk 3 above)
Comment (hheyhhay) [reply src/scripts.js:42]: Will do :)

**Pull request merged**